### DISCUSSION TITLE
Public Rendezvous Server Status Page?

### DISCUSSION THREAD

**Discussion by stephen-hill** [rustdesk/rustdesk#6996] [General]
Hi

Is there a status page for the availability of the public rendezvous servers?

I'm having problems connecting to clients today and I'm wondering if it's because of the rendezvous servers. A status page would be very helpful in debugging the problem.

Many Thanks
Stephen

**Comment by dsavic11** [comment]
Having the same problem today. Fully support your request for a status page for the availability of the public rendezvous servers.

**Comment by royaltongue** [comment]
Also having sporadic connection issues today. If I spam connecting, it'll go through and then stay connected, but mostly it says the ID doesn't exist and I'm not connected to the RustDesk network
